js/data/custom_store.js

```diff
@@ -260,7 +260,7 @@ var CustomStore = Store.inherit({
          * @type_function_param1_field8 requireTotalCount:boolean
          * @type_function_param1_field9 searchValue:object
          * @type_function_param1_field10 searchOperation:string
-         * @type_function_param1_field11 searchExpr:getter|array
+         * @type_function_param1_field11 searchExpr:getter|Array<string>
```

Comment:
IIRC, the correct value is: 
`getter|Array<getter>`

/cc @AlekseyMartynov

Reply:
That's right. See https://github.com/DevExpress/DevExtreme/blob/9cb3052fa31bca1021805b79a16e08e37a76e4bc/js/data/data_source/data_source.js#L891-L893